app/electron/main.ts

```diff
@@ -258,6 +258,24 @@ function setMenu(i18n: I18n) {
   Menu.setApplicationMenu(menu);
 }
 
+function calculateWindowSize() {
+  const { width, height } = screen.getPrimaryDisplay().workAreaSize;
+  const [maxWidth, maxHeight] = [1920, 1080];
+  const pixelRatio = screen.getPrimaryDisplay().scaleFactor;
```

Comment:
scaleFactor is a Number which can be a float. So a 1.5x screen should be returned here as `1.5`.

So the case statement below should be changed into a series of ranges.